webdriver/tests/websockets/connect.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import asyncio
+import websockets
+
+# classic session to enable bidi capability manually
+@pytest.mark.asyncio
+@pytest.mark.capabilities({"webSocketUrl": True})
+async def test_websocket_url_connect(session):
+    websocket_url = session.capabilities["webSocketUrl"]
+    async with websockets.connect(websocket_url) as websocket:
+        await websocket.send("Hello world!")
+        await websocket.close()
+
+# bidi session following classic session to test session
+# recreation with bidi true in session fixture.
+# Close websocket at the end.
+@pytest.mark.asyncio
+@pytest.mark.bidi(True)
+async def test_bidi_session_1(session):
+    await session.websocket_transport.send("test_bidi_session_1")
+    await session.websocket_transport.close()
+
+# bidi session following a bidi session with the same capabilities
```

Comment:
This makes the test dependent on a strict order, and `test_bidi_session_1` being enabled. To get rid of this all the code should be put into a single test.

Reply:
One of the reasons for the tests is testing the behavior from re-using websocket_transport between tests with/without capability change. I don't think I can achieve that with a single test.

Reply:
We might need a similar `new_session` fixture for BiDi so that you can create multiple new sessions with different capabilities within a single test.